liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/BasicSupport.java

```diff
@@ -374,13 +376,43 @@ private File checkServerHome(final File archive) throws IOException,
 
         return dir.getCanonicalFile();
     }
+    
+    private InitLog initLog = new InitLog();
+    private enum MessageType {INFO, DEBUG};
+    private class InitLog {
+    	
+    	private List<MessageType> msgTypes = new ArrayList<MessageType>();
+    	private List<String> messages = new ArrayList<String>();
+
+		public void logInitMessages() {
```

Comment:
Perhaps this class should have a boolean that indicates whether or not the initLog messages have been logged to the real log? And should the messages get cleared out of this class once they have been logged to the real log?

Reply:
@cherylking , I added the clear in the new commit.   I'm not sure adding another boolean would do anything more in particular but please see what you think.  Thx.